Revert "MIMXRT1050_EVK: Add IAR support in the exporter"

0xc0170 opened the pull request:
### Description

Reverts ARMmbed/mbed-os#9496

Reason: MCU is only available in IAR 8.22 and higher. Please see 9610 for more details

### Pull request type

    [x] Fix
    [ ] Refactor
    [ ] Target update
    [ ] Functionality change
    [ ] Docs update
    [ ] Test update
    [ ] Breaking change

0xc0170 committed:
Revert "MIMXRT1050_EVK: Add IAR support in the exporter"

ciarmcom commented:
@0xc0170, thank you for your changes.
@ARMmbed/mbed-os-tools @ARMmbed/mbed-os-maintainers please review.

0xc0170 commented:
To fix the nightly, I started CI

adbridge commented:
@0xc0170 Please add the missing template header ;-)

bridadan reviewed (approved):
The changes themselves are fine. Just be sure we don't forget to put this back once we have IAR 8 support!

0xc0170 commented:
> @0xc0170 Please add the missing template header ;-)

Not sure how that happen , fixed now

It could be if revert via github, it wont catch the template? or I just removed it when typing
